appscale/tools/appscale_tools.py

```diff
@@ -51,6 +57,12 @@ def async_layout_upgrade(ip, keyname, script, error_bucket, verbose=False):
     error_bucket.put(ssh_error)
 
 
+MIN_FREE_DISK_DB = 40.0
```

Comment:
@cdonati you're right. But maybe it makes sense to refactor AppController first? It doesn't look like a critical functionality, so imho it can wait for REST-like architecture of AppController. If I don't move warning-functionality to AppController, do you want me to remove it from tools?